AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginIntegrationTests/GraphQL/GraphQLWithLambdaAuthIntegrationTests/GraphQLWithLambdaAuthIntegrationTests.swift

```diff
@@ -238,6 +238,10 @@ class GraphQLWithLambdaAuthIntegrationTests: XCTestCase {
 
 // MARK: - API Auth provider
 private class CustomTokenProvider: AmplifyFunctionAuthProvider {
+    func getUserPoolAccessToken() async throws -> String {
+        return "custom-lambda-token"
+    }
+    
```

Comment:
This will most likely change as well, from the discussion with @ameter , AmplifyFunctionAuthProvider is a protocol used by AWS Lambda Auth use cases and is actually just a token that they will implement the code that retreives and returns it. It is not related to Cognito User Pools